src/app/shared/pipes/creation-date/creation-date.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import {formatDate} from "@angular/common";
+
+@Pipe({
+  name: 'creationDate'
+})
+export class CreationDatePipe implements PipeTransform {
+
+  transform(value: Date, ...args: unknown[]): string {
```

Comment:
```suggestion
  transform(value: Date): string {
```
Let's remove extra arguments if we don't use them